en/docs/design-the-project/explore-the-architecture-view.md

```diff
@@ -125,9 +119,30 @@ If these data types are Ballerina records, click on them to access the compositi
 
 <img src="https://wso2.com/ballerina/vscode/docs/img/visual-programming/architecture-view/architecture-diagram/level-two-to-type-composition.gif" class="cInlineImage-full"/>
 
-### Type Diagram
+### Type diagram
+
+The type diagram provides a comprehensive view of all the Ballerina record types defined in the project workspace and their associations. This can be accessed via the **Architecture View** command, as shown below.
+
+<img src="https://wso2.com/ballerina/vscode/docs/img/visual-programming/architecture-view/type-diagram/type-diagram.gif" class="cInlineImage-full"/>
+
+The following is a snippet of the type diagram generated for the [Online Boutique Microservices demo](https://github.com/ballerina-guides/gcp-microservices-demo). 
+
+<img src="https://wso2.com/ballerina/vscode/docs/img/visual-programming/architecture-view/type-diagram/gcp-type-diagram.png" class="cInlineImage-full"/>
+
+!!! Tip 
+    The multiplicities of the associations are represented on either side of the connector, while the type inclusions are differentiated using a directed link (represents inheritance).
+
+### Composition diagram
+
+You can also view the **composition** of an individual record type. Use either of the methods below to access these composition diagrams.
+
+1. Click the record types that are included in the request/response bodies of the service invocations shown in the [level 2 service diagram](https://wso2.com/ballerina/vscode/docs/visual-programming/architecture-view/architecture-diagram/#service-diagram-level-2).
```

Comment:
Is it correct to have link from live site as the urls are going to change?

Reply:
Fixed.